src/components/Post/Icon-like.js

```diff
@@ -1,24 +1,101 @@
 import styled from "styled-components";
+import { AiFillHeart } from "react-icons/ai";
 import { TbHeart } from "react-icons/tb";
+import ReactTooltip from "react-tooltip";
```

Comment:
This dependency import is causing the application to crash, as react-tooltip librarie is not included as one of the dependencies on package.json. Install it with "npm i react-tooltip" or include - "react-tooltip": "^4.4.3" - on package.json and it should work fine

Reply:
Good observation, thank u very much.